internal/pkg/describe/status.go

```diff
@@ -341,6 +336,391 @@ func (a *apprunnerServiceStatus) HumanString() string {
+			(int)(s.Service.DesiredCount) - (int)(s.Service.RunningCount),
+		},
+		[]string{color.Green.Sprint("■"), color.Grey.Sprint("□")})
+	stringSummary := fmt.Sprintf("%d/%d desired tasks are running", s.Service.RunningCount, s.Service.DesiredCount)
+	fmt.Fprintf(writer, "  %s\t%s\t%s\n", header, bar, stringSummary)
+
+	// Write summary of task definition revision.
+	desiredTaskDefVersion, err := awsecs.TaskDefinitionVersion(s.Service.TaskDefinition)
+	if err == nil && s.Service.RunningCount > 0 {
+		header := "Deployment"
+		data := s.taskDefinitionRevisionData()
+		bar := summaryBar(
+			[]int{
+				data[desiredTaskDefVersion],
+				(int)(s.Service.RunningCount) - data[desiredTaskDefVersion],
+			},
+			[]string{color.Green.Sprint("■"), color.Grey.Sprint("□")})
+		stringSummary := fmt.Sprintf("%d/%d running task definition version %d (desired)",
+			data[desiredTaskDefVersion],
+			s.Service.RunningCount,
+			desiredTaskDefVersion)
+		fmt.Fprintf(writer, "  %s\t%s\t%s\n", header, bar, stringSummary)
+	}
+
+	// Write summary of HTTP health and container health.
+	if s.shouldShowHealthSummary() {
+		header := "Healthy"
+		if s.shouldShowHTTPHealth() {
+			healthyCount := s.healthyHTTPTasksCount()
+			bar := summaryBar(
+				[]int{
+					healthyCount,
+					(int)(s.Service.RunningCount) - healthyCount,
+				},
+				[]string{color.Green.Sprint("■"), color.Grey.Sprint("□")})
+			stringSummary := fmt.Sprintf("%d/%d passes HTTP health checks", healthyCount, s.Service.RunningCount)
+			fmt.Fprintf(writer, "  %s\t%s\t%s\n", header, bar, stringSummary)
+			header = ""
+		}
+
+		if s.shouldShowContainerHealth() {
+			healthyCount, _, _ := s.containerHealthData()
+			bar := summaryBar(
+				[]int{
+					healthyCount,
+					(int)(s.Service.RunningCount) - healthyCount,
+				},
+				[]string{color.Green.Sprint("■"), color.Grey.Sprint("□")})
+			stringSummary := fmt.Sprintf("%d/%d passes container health checks", healthyCount, s.Service.RunningCount)
+
+			fmt.Fprintf(writer, "  %s\t%s\t%s\n", header, bar, stringSummary)
+
+		}
+	}
+
+	// Write summary of capacity providers.
+	if s.shouldShowCapacityProvider() {
+		header := "Capacity Provider"
+		fargate, spot, unset := s.capacityProviderData()
+		bar := summaryBar([]int{fargate, spot, unset}, []string{color.Blue.Sprint("f"), color.Magenta.Sprint("s"), color.Grey.Sprint("d")})
+		var cpSummaries []string
+		if fargate != 0 {
+			cpSummaries = append(cpSummaries, fmt.Sprintf("%d/%d on Fargate", fargate, s.Service.RunningCount))
+		}
+		if spot != 0 {
+			cpSummaries = append(cpSummaries, fmt.Sprintf("%d/%d on Fargate Spot", spot, s.Service.RunningCount))
+		}
+		if unset != 0 {
+			cpSummaries = append(cpSummaries, fmt.Sprintf("%d/%d not scaled by capacity provider", unset, s.Service.RunningCount))
+		}
+		fmt.Fprintf(writer, "  %s\t%s\t%s\n", header, bar, strings.Join(cpSummaries, ", "))
+	}
+	writer.Flush()
+}
+
+func (s *ecsServiceStatus) writeStoppedTasks(writer *tabwriter.Writer) {
```

Comment:
nit: for these in case we ever change the writer (although unlikely :P)
```suggestion
func (s *ecsServiceStatus) writeStoppedTasks(writer io.Writer) {
```

Reply:
addressed!